src/Rosbag2.ts

```diff
@@ -26,10 +26,21 @@ for (const ros1Datatype in commonDefs) {
   ROS2_DEFINITIONS_ARRAY.push(msgdef);
   ROS2_TO_DEFINITIONS.set(ros2Datatype, msgdef);
 }
-for (const ros1Datatype in foxgloveDefs) {
-  const ros2Datatype = ros1Datatype.replace("_msgs/", "_msgs/msg/");
+
+const imageMarkerArray: RosMsgDefinition = {
+  name: "foxglove_msgs/ImageMarkerArray",
+  definitions: [
+    { type: "visualization_msgs/ImageMarker", isArray: true, name: "markers", isComplex: true },
+  ],
+};
+ROS2_DEFINITIONS_ARRAY.push(imageMarkerArray);
+ROS2_TO_DEFINITIONS.set("foxglove_msgs/msg/ImageMarkerArray", imageMarkerArray);
+
+for (const schema of Object.values(foxgloveMessageSchemas)) {
+  const { qualifiedRosName, fields } = generateRosMsgDefinition(schema, { rosVersion: 2 });
+  const ros2Datatype = qualifiedRosName.replace("_msgs/", "_msgs/msg/");
```

Comment:
If `rosVersion: 2` is being passed to `generateRosMsgDefinition()`, shouldn't it return the correct qualifiedRosName already?

Reply:
Maybe we need to add a separate field, because ROS 2 uses both the `foxglove_msgs/X` and the `foxglove_msgs/msg/X` version in different ways.